### DISCUSSION TITLE
Support base target to be _parent

### DISCUSSION THREAD

**Discussion by xXCalleManXx** [vercel/next.js#47915] [Ideas]
### Goals

1. metadataBase should also support a string value instead of only URL object

### Non-Goals

1.
2.
3.

### Background

Using the new APP directory i have created a button with a link. This button is used inside an IFRAME on another webpage. When clicking the link inside the iframe, the parent window must navigate to the target link.

### Proposal

`metadataBase` should support values such as `_blank`, `_self`, `_parent`, `_top` as described here: https://www.w3schools.com/tags/att_base_target.asp

**Comment by lawrencejob** [comment]
Hi. I think you're trying to use `<base>` (as linked to at w3schools). [Using base is not supported](https://beta.nextjs.org/docs/api-reference/metadata#unsupported-metadata) in the new metadata API. The `metadataBase` key is for a different purpose. [You can read about it more in the docs.](https://beta.nextjs.org/docs/api-reference/metadata#metadatabase)

**Reply by horaklukas** [reply]
Hi @lawrencejob don't you know if there is any way to inject `base` tag into the head? I use it to set default `target` for all links on the page when it's a page that should be embedded via iframe. 

* Unfortunately `next/head` doesn't work in `app` dir as an escape hatch
* I can't put it into the head directly because it's in deeper in routes hirearchy and root layout is usual
 ```tsx
 export default function RootLayout({ children }: PropsWithChildren) {
  return (
    <html lang="en">
      <body className={inter.className}>
        <StyledComponentsRegistry>{children}</StyledComponentsRegistry>
      </body>
    </html>
  );
}
```
